models/articles.models.js

```diff
@@ -26,6 +27,18 @@ function fetchArticles() {
     .then(response => {
         return response.rows;
     })
+} else {
+    return db
+    .query(`SELECT * FROM articles WHERE articles.topic = $1;`,
```

Comment:
When adding the topic query the results should be of the same format as without it. 

It should be the same columns you specified in the above query and include the comment_count still